### PR TITLE
Fix hudi scan type

### PR DESCRIPTION
**Pull request by wsjz**
# Proposed changes

Obviously a little mistake

## Problem Summary:

image

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (No Need)
3. Has document been added or modified: (No Need)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

## Further comments

Maybe have a better judge in the future

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/planner/BrokerScanNode.java [hunk 1]
```diff
@@ -152,7 +152,7 @@ public BrokerScanNode(PlanNodeId id, TupleDescriptor destTupleDesc, String planN
         this.fileStatusesList = fileStatusesList;
         this.filesAdded = filesAdded;
         if (ConnectContext.get() != null) {
-            this.userIdentity = ConnectContext.get().getUserIdentity();
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/planner/BrokerScanNode.java:155]: This has been changed in master branch. May be you need to rebase the code

**Review by morningman — COMMENTED** [review]
Please correct the PR title, you can refer to 
http://doris.incubator.apache.org/zh-CN/community/how-to-contribute/commit-format-specification.html

File: fe/fe-core/src/main/java/org/apache/doris/planner/BrokerScanNode.java (hunk 1 above)
Comment (wsjz) [reply fe/fe-core/src/main/java/org/apache/doris/planner/BrokerScanNode.java:155]: OK,  I will close it and create new PR:  this https://github.com/apache/incubator-doris/pull/9984

**Comment by wsjz** [comment]
use this instead: https://github.com/apache/incubator-doris/pull/9984

**Pull request closed without merging**